apps/cli/src/commands/build.ts

```diff
@@ -132,39 +132,67 @@ export default class BuildApplication extends Command {
         return info;
     }
 
-    private async exportImageTar(
+    // creates docker image tar
+    private async exportImageTar(image: string, path: string): Promise<void> {
+        // delete tar if exists
+        if (fs.existsSync(path)) fs.unlinkSync(path);
+
+        // save container image tar
+        await execa("docker", ["image", "save", image, "--output", path], {
+            stdio: "inherit",
+        });
+    }
+
+    // creates the rootfs tyar from the image
+    private async createRootfsTar(
         image: string,
-        tarPath: string,
+        container: string,
     ): Promise<void> {
-        // create container
-        const { stdout: cid } = await execa("docker", [
-            "container",
-            "create",
-            "--platform",
-            "linux/riscv64",
-            image,
-        ]);
+        //create rootfs tar
+        await execa(
+            "docker",
+            [
+                "container",
+                "exec",
+                container,
+                "undocker",
+                `/tmp/${SUNODO_DEFAULT_DOCKER_TAR_PATH}`,
+                `/tmp/${SUNODO_DEFAULT_TAR_PATH}`,
+            ],
+            { stdio: "inherit" },
+        );
 
-        // export container rootfs to tar
-        await execa("docker", ["export", "-o", tarPath, cid]);
+        // delete tar if exists
+        if (fs.existsSync(SUNODO_DEFAULT_DOCKER_TAR_PATH))
+            fs.unlinkSync(SUNODO_DEFAULT_DOCKER_TAR_PATH);
+    }
 
-        // remove container
-        await execa("docker", ["rm", cid]);
+    private async exportRootfsTar(
+        container: string,
+        path: string,
+    ): Promise<void> {
+        await execa(
+            "docker",
+            [
+                "container",
+                "cp",
+                `${container}:/tmp/${SUNODO_DEFAULT_TAR_PATH}`,
+                path,
+            ],
+            { stdio: "inherit" },
+        );
     }
 
-    private async getBuildContainer(
-        image: string,
-        tarPath: string,
-    ): Promise<string> {
+    private async getBuildContainer(image: string): Promise<string> {
         const { stdout: cid } = await execa("docker", [
             "container",
             "run",
             "--detach",
             "--init",
-            `--volume=${tarPath}:/tmp/${SUNODO_DEFAULT_TAR_PATH}`,
+            `--volume=./${SUNODO_DEFAULT_DOCKER_TAR_PATH}:/tmp/${SUNODO_DEFAULT_DOCKER_TAR_PATH}`,
```

Comment:
I did not understand why we need this volume mapped, if the `docker-image.tar` does not need to be in the host filesystem